extensions/ql-vscode/src/extension.ts

```diff
@@ -493,6 +494,29 @@ async function activateWithInstalledDistribution(
     }
   }
 
+  async function previewQueryHelp(
+    selectedQuery: Uri
+  ): Promise<void> {
+    const path = selectedQuery ? selectedQuery.path : window.activeTextEditor!.document.uri.fsPath;
```

Comment:
If this command is called from the context menu on the editor, I would expect `selectedQuery` to be exactly `window.activeTextEditor.document.uri`. Is that what you observed while testing this?

Reply:
Yes, as far as I've observed they are equal. `JSON.stringify(selectedQuery) === JSON.stringify(window.activeTextEditor.document.uri)` also evaluates to `true` in the cases I've tested

Reply:
In that case I think it should be enough to use `selectedQuery.fsPath`.

Reply:
Just `path = selectedQuery.fsPath`

Reply:
`path = selectedQuery.fsPath` fails for me when I run the command from the palette